mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageDAO.java

```diff
@@ -197,7 +197,7 @@ private ByteBuffer toByteBuffer(InputStream stream) throws IOException {
             ids -> FluentFutureStream.of(
                 ids.stream()
                     .map(id -> retrieveRow(id, fetchType)
-                        .thenApply(resultSet ->
+                        .thenApply((ResultSet resultSet) ->
```

Comment:
Please reread my comment and your commit. They are not related. I see no comment on your commit explaining why we add type